### PR TITLE
Ace/ctrd

### PR DESCRIPTION
**Pull request by alexeldeib**

### CODE REVIEW

File: vhdbuilder/packer/install-dependencies.sh [hunk 1]
```diff
@@ -180,7 +180,7 @@ installBpftrace
 echo "  - bpftrace" >> ${VHD_LOGS_FILEPATH}
 
 if [[ $OS == $UBUNTU_OS_NAME && $(isARM64) != 1 ]]; then  # no ARM64 SKU with GPU now
-installGPUDrivers
+downloadGPUDrivers
```
Comment (alexeldeib) [inline vhdbuilder/packer/install-dependencies.sh:183]: note to self: might need to install nvidia-docker2 here.

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh [hunk 2]
```diff
@@ -232,4 +215,56 @@ cleanUpGPUDrivers() {
...
 
+blacklistNouveau() {
+    tee /etc/modprobe.d/blacklist-nouveau.conf > /dev/null <<EOF
+blacklist nouveau
+options nouveau modeset=0
+EOF
+    retrycmd_if_failure_no_stats 120 5 25 update-initramfs -u || exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
+}
+
+addNvidiaAptRepo() {
```
Comment (alexeldeib) [inline parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:226]: TODO: we can avoid doing this at runtime by checking for the cached versions of the files before invoking the install commands.

probably won't do it in this pr tho

**Review by ganeshkumar5699 — APPROVED** [review]
Thanks Ace!

File: vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh [hunk 3]
```diff
@@ -70,15 +70,6 @@ configGPUDrivers() {
     retrycmd_if_failure_no_stats 120 5 25 update-initramfs -u || exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
     wait_for_apt_locks
     retrycmd_if_failure 30 5 3600 apt-get -o Dpkg::Options::="--force-confold" install -y nvidia-container-runtime="${NVIDIA_CONTAINER_RUNTIME_VERSION}" || exit $ERR_GPU_DRIVERS_INSTALL_TIMEOUT
-    tmpDir=$GPU_DEST/tmp
```
Comment (alexeldeib) [inline vhdbuilder/scripts/linux/ubuntu/tool_installs_ubuntu.sh:73]: boggles my mind we have a separate file which overrides funcs installed from cse_install. at least it still works okay for vhd install. but TODO: get rid of the duplication.

**Pull request merged**